data/test/scenarios/test_move_unit_to_occupied_hex.cfg

```diff
@@ -0,0 +1,55 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [move_unit]to_location
+##
+# Actions:
+# Try to move Alice to the hex that Bob already occupies.
+##
+# Expected end state:
+# Bob has not moved and both units are still on the map.
+# Alice is adjacent to Bob.
+#
+# The main "test_move_unit" includes a check that moving to an occupied
+# location by x,y coordinates doesn't overwrite the existing unit. This
+# tests the same, using a location_id.
+#####
+{GENERIC_UNIT_TEST "test_move_unit_to_occupied_location" (
+    [event]
+        name=start
+        {ASSERT (
+            [have_unit]
+                id=alice
+                x=7
+                y=3
+            [/have_unit]
+            [have_unit]
+                id=bob
+                x=13
+                y=3
+            [/have_unit]
+        )}
+
+        [move_unit]
+            id=alice
+            to_location=2
```

Comment:
This is a location that has already been defined, I assume?

Reply:
Numerical location ids `1`, `2`, etc correspond to the starting location for that side.